_posts/2013-09-26-authentication.markdown

```diff
@@ -35,6 +39,8 @@ Here is the sample application information we'll use throughout:
   redirect_url: https://example.com/callback
 ```
 
+The redirect_url is sometimes referred to as a callback URI and the id can also be called a client_id. The id and secret will be generated when you supply a redirect_url to Gusto. (Note: when you receive your API keys for Gusto, we will include an API Token. This API token is only used when creating a company through the API.)
```

Comment:
The block above this is actually incorrect and says `redirect_url` instead of `redirect_uri`. 😬 While you're making these changes, would you mind updating mentions of redirect_url to redirect_uri in this file?

I would also put any mentions of redirect_uri or client_id in backticks (`) similar to the URLs above.

Reply:
done. thank you!